WalletWasabi/TorSocks5/TorProcessManager.cs

```diff
@@ -145,7 +145,7 @@ public void Start(bool ensureRunning, string dataDir)
 						}
 						else // Linux and OSX
 						{
-							string runTorCmd = $"LD_LIBRARY_PATH=$LD_LIBRARY_PATH:={torDir}/Tor && export LD_LIBRARY_PATH && cd {torDir}/Tor && ./tor {torArguments}";
+							string runTorCmd = $"LD_LIBRARY_PATH=$LD_LIBRARY_PATH:={torDir}/Tor && export LD_LIBRARY_PATH && cd '{torDir}/Tor' && ./tor {torArguments}";
```

Comment:
Are you sure about this? You cannot create a user name that contains spaces. Better be safe but if this works shouldn't you add quotes to LD_LIBRARY_PATH:={torDir}/Tor too?

Reply:
> Are you sure about this?

Paths should be always escaped. And single quotes do not allow to resolve variables in Bash so I think it's ok.

> Are you sure about this?

`"LD_LIBRARY_PATH=$LD_LIBRARY_PATH:={torDir}/Tor` I'm not familiar with this syntax but one should add quotes there too.

> You cannot create a user name that contains spaces. 

Well, I guess that's not bulletproof argument. Maybe some distro allows that, and even if not why should one worry that somebody will do something crazy with their filesystem ... Quotes give peace of mind that shell process that path as single argument. That's it.

Reply:
However, the quotes are not consistent it is working as it is. IMO it is OK.

`2020-07-27 14:57:34 INFO	TorProcessManager (150)	Started Tor process with shell command: LD_LIBRARY_PATH=$LD_LIBRARY_PATH:='/home/molnard/.walletwasabi/client/tor/Tor' && export LD_LIBRARY_PATH && cd '/home/molnard/.walletwasabi/client/tor/Tor' && ./tor --SOCKSPort 127.0.0.1:9050 --DataDirectory "/home/molnard/.walletwasabi/client/tordata" --GeoIPFile "/home/molnard/.walletwasabi/client/tor/Data/Tor/geoip" GeoIPv6File "/home/molnard/.walletwasabi/client/tor/Data/Tor/geoip6" --Log "notice file /home/molnard/.walletwasabi/client/TorLogs.txt".`

Reply:
IMO it's good enough at this point. 

It can be certainly improved further, https://github.com/zkSNACKs/WalletWasabi/pull/4057 is start.